readme.txt

```diff
@@ -0,0 +1,63 @@
+=== Sensei LMS Certificates ===
+Contributors: automattic, alexsanford1, donnapep, jakeom, gkaragia, renatho, yscik
+Tags: certificates, course certificate, sensei lms
+Requires at least: 4.9
+Tested up to: 5.3
+Requires PHP: 5.6
+Stable tag: 2.0.2
+License: GPLv2+
+License URI: http://www.gnu.org/licenses/gpl-2.0.html
+
+Award your students with a certificate of completion and a sense of accomplishment after finishing a course.
+
+== Description ==
+
+= Award your students with a certificate of completion for their completed courses. =
+
+There’s no feeling quite like it. You’ve studied hard, taken your test and passed (with flying colours, I’m sure)! After all that effort, you’re finally at the end of the course. You want to share this accolade with everyone. Print out your certificate!
+
+With Sensei LMS Certificates, your students are awarded a certificate for each course they complete through your online school. These certificates can be downloaded, printed and framed on their office wall, or stuck up on the fridge for the entire family to enjoy. If you’d like to make certificates public for all to view, that’s available as well.
+
+= Customise the certificate design =
+
+Sensei LMS Certificates includes an advanced certificate design system. Through this system, you can upload your own background image, place and style the size, typeface and colour of the various pieces of text on the certificate and truly create a unique design best suited to your online school.
+
+If you don’t fancy yourself as a designer, or would prefer to get up and running in not much time, Sensei LMS Certificates includes a stylish default certificate design, suited to any genre of online school.
+
+= We already have many students who have completed courses! Can they get certificates as well? =
+
+They certainly can! When activating Sensei LMS Certificates, you will be prompted to generate certificates for each student who has already completed a course. Thankfully, this can be done, for all students, with the click of a button.
+
+= Increase student retention =
+
+We all enjoy receiving rewards. If your students complete a single course, and receive their certificate for the hard work they’ve done, they will be statistically more likely to continue on to a further course, in order to re-live the rewarding experience of receiving a certificate for their efforts.
+
+We look forward to seeing your students light up with excitement when they receive their certificates for all the hard work they’ve done in completing your courses.
+
+= Key Features =
+
+* Easily-customised certificate design
+* Lean code, to ensure a pleasant experience
+* Intuitive integration with Sensei
```

Comment:
Should be "Sensei LMS" now.

Reply:
Fixed here: https://github.com/woocommerce/sensei-certificates/pull/215/commits/cc69c740ae7632376e07b006adb8db2ace42779f